src/actions/ServiceGraphDataActions.ts

```diff
@@ -38,7 +38,7 @@ export const ServiceGraphDataActions = {
       const restParams = { duration: duration + 's' };
       API.getGraphElements(namespace, restParams).then(
         response => {
-          const responseData = response['data'];
+          const responseData: any = response['data'];
```

Comment:
Just for me, as I see in the code that the use of any is optional. What are the benefits to add the any here ?

Reply:
It isn't optional in this case because I've changed API functions to return Promise<Response<T>> ; when T is unknown (as here) typescript decides it's an empty object `{}`, and here I got compile errors so I had either to declare a proper type (but I'm lacking that knowledge) or cast to `any`